### PR TITLE
Add Windows Server 2025 Updates

### PR DESCRIPTION
**Pull request by mrhoads**
This PR updates text and screenshots to reflect the use of Windows Server 2025 and adds a short description of the new management features for Windows Server 2025 as an optional step.

### CODE REVIEW

File: docs/azure_jumpstart_arcbox/ITPro/arch_itpro.png
Comment (janegilring) [inline docs/azure_jumpstart_arcbox/ITPro/arch_itpro.png:1]: See if we can export an high-dpi version of this image

**Review by janegilring — CHANGES_REQUESTED** [review]
LGTM besides the resolution for the architecture diagram I pointed out

File: docs/azure_jumpstart_arcbox/ITPro/arch_itpro.png
Comment (mrhoads) [reply docs/azure_jumpstart_arcbox/ITPro/arch_itpro.png:1]: new architecture diagram has been exported in high-dpi

**Review by janegilring — APPROVED** [review]
LGTM

**Pull request merged**